schemas/json/block.json

```diff
@@ -675,6 +675,29 @@
 							}
 						}
 					]
+				},
+				"interactivity": {
+					"description": "Indicates if the block is using Interactivity API features.",
+					"oneOf": [
+						{
+							"type": "boolean"
+						},
+						{
+							"type": "object",
+							"properties": {
+								"clientNavigation": {
+									"type": "boolean",
+									"description": "Indicates if a block is compatible with Interactivity API client side navigation.\n\nSet as false if the block is using vanilla JS or another framework to handle interactions.",
```

Comment:
```suggestion
									"description": "Indicates if a block is compatible with Interactivity API client-side navigation.\n\nSet it to true only if the block is not interactive or if it is interactive using the Interactivity API. Set it as false if the block is interactive but it is using vanilla JS, jQuery or another JS framework/library different than the Interactivity API to handle interactions.",
```

Reply:
We could get help from a native speaker here. This is complex to understand 🙂

Reply:
Ping @sirreal, @ndiego or @ryanwelcher

Reply:
Tricky 🙂 I've read the descriptions and have questions:

`interactivity`: boolean|object - What does `true` mean here? Enable all the features?

`true === { clientNavigation: true, interactive: true }`
`(omission) === false === {} `

Should we include the default value for `interactivity` (which I think is `false`)?

Can we have `clientNavigation` without `interactive`, or does `clientNavigation` require `interactive` (if they're dependent we can reflect that in the schema itself instead of describing it).

---

What does "interactive" mean here?

> Set it to true only if the block is not interactive or if it is interactive using the Interactivity API
> Set it to false if the block is interactive but uses vanilla JS… other than the Interactivity API

Interactive sounds like `interactivity.interactive`, but are we talking about that? Or more generally that a block has client-side JavaScript to make it _do things_.

Reply:
> interactivity: boolean|object - What does true mean here? Enable all the features?

Yes

> (omission) === false === {} 

I think in this case would be:
```
{
  clientNavigation: false,
  interactive: false
}
```

> Should we include the default value for interactivity (which I think is false)?

Sure, done.

> Interactive sounds like interactivity.interactive, but are we talking about that? Or more generally that a block has client-side JavaScript to make it do things.

Interactive means that a block is using the Interactivity API as alternative of JavaScript made things.

If the developer is using JS or other framework. `clientNavigation` should be false. `interactive` in this case would not affect anything on the code. And for that, reason, all options are opt-out. (usually in this case, the user will not update or know about this value)